### ISSUE TITLE
Icons overlay

### ISSUE THREAD

**Issue by bardware** [cfeclipse/cfeclipse#91]
There's an error icon but the first message does not read `ERROR`

![grafik](https://user-images.githubusercontent.com/802702/31012783-6be36e6e-a513-11e7-8e5f-39bf15d74e68.png)
![grafik](https://user-images.githubusercontent.com/802702/31012798-79726440-a513-11e7-94b4-217efef5cf51.png)

**Comment by denuno** [comment]
Are you still seeing this with the latest from dev?

**Comment by bardware** [comment]
Still there
![image](https://user-images.githubusercontent.com/802702/34789479-b5e55686-f63e-11e7-8fa8-22adb20076d8.png)

**Comment by bardware** [comment]
Sorry it took so long to answer.
Today, I finally set up a new clean eclipse from Runtime Binary 4.7.2. I added a single Update Site http://cfeclipse.org/update-dev/ and installed what was available.
When I finally setup CFLint for the project it took its time and came up with overlapping icons. :(

![grafik](https://user-images.githubusercontent.com/802702/36214433-c4c5f396-11a9-11e8-9dc0-4377d64a8a10.png)
